### PR TITLE
Test integration api views

### PR DESCRIPTION
**Pull request by Jakub89**
I move some tests from test_integration_report_computed_task.py that were only checking api_view.py to utils.tests. And add one more in a process.

### CODE REVIEW

File: concent_api/utils/tests/test_integration_api_view.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        task_to_compute.compute_task_def['deadline'] = int(dateutil.parser.parse("2017-12-01 11:00:00").timestamp())
+
+        serialized_task_to_compute      = dump(task_to_compute,             PROVIDER_PRIVATE_KEY,   REQUESTOR_PUBLIC_KEY)
+        deserialized_task_to_compute    = load(serialized_task_to_compute,  REQUESTOR_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY, check_time = False)
+
+        force_report_computed_task = message.ForceReportComputedTask(
+            timestamp               = int(dateutil.parser.parse("2017-12-01 10:59:00").timestamp()),
+        )
+        force_report_computed_task.task_to_compute = deserialized_task_to_compute
+        serialized_force_report_computed_task = dump(force_report_computed_task, PROVIDER_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
```
Comment (kbeker) [inline concent_api/utils/tests/test_integration_api_view.py]: Style:
```
force_report_computed_task.task_to_compute = deserialized_task_to_compute
serialized_force_report_computed_task      = dump(force_report_computed_task, PROVIDER_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
```

File: concent_api/utils/tests/test_integration_api_view.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+        # STEP 1: Provider forces computed task report via Concent
+
+        task_to_compute = message.TaskToCompute(
+            timestamp   = int(dateutil.parser.parse("2017-12-01 10:00:00").timestamp()),
+        )
+
+        task_to_compute.compute_task_def = message.ComputeTaskDef()
+        task_to_compute.compute_task_def['task_id'] = 1
+        task_to_compute.compute_task_def['deadline'] = int(dateutil.parser.parse("2017-12-01 11:00:00").timestamp())
```
Comment (kbeker) [inline concent_api/utils/tests/test_integration_api_view.py]: Style

File: concent_api/utils/tests/test_integration_api_view.py [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+        # STEP 1: Provider forces computed task report via Concent
+
+        task_to_compute = message.TaskToCompute(
+            timestamp = int(dateutil.parser.parse("2017-12-01 10:00:00").timestamp()),
+        )
+
+        task_to_compute.compute_task_def = message.ComputeTaskDef()
+        task_to_compute.compute_task_def['task_id']     = 1
+        task_to_compute.compute_task_def['deadline']    = int(dateutil.parser.parse("2017-12-01 11:00:00").timestamp())
```
Comment (kbeker) [inline concent_api/utils/tests/test_integration_api_view.py]: Style

File: concent_api/utils/tests/test_integration_api_view.py [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
```
Comment (kbeker) [inline concent_api/utils/tests/test_integration_api_view.py]: Style

File: concent_api/utils/tests/test_integration_api_view.py [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+        task_to_compute.compute_task_def['deadline'] = int(dateutil.parser.parse("2017-12-01 11:00:00").timestamp())
+
+        serialized_task_to_compute      = dump(task_to_compute,             PROVIDER_PRIVATE_KEY,   REQUESTOR_PUBLIC_KEY)
+        deserialized_task_to_compute    = load(serialized_task_to_compute,  REQUESTOR_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY, check_time = False)
+
+        force_report_computed_task = message.ForceReportComputedTask(
+            timestamp               = int(dateutil.parser.parse("2017-12-01 10:59:00").timestamp()),
+        )
+        force_report_computed_task.task_to_compute = deserialized_task_to_compute
+        serialized_force_report_computed_task = dump(force_report_computed_task, PROVIDER_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
```
Comment (kbeker) [inline concent_api/utils/tests/test_integration_api_view.py]: Style

File: concent_api/utils/tests/test_integration_api_view.py [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+        # STEP 1: Provider forces computed task report via Concent
+
+        task_to_compute = message.TaskToCompute(
+            timestamp   = int(dateutil.parser.parse("2017-12-01 10:00:00").timestamp()),
+        )
+
+        task_to_compute.compute_task_def = message.ComputeTaskDef()
+        task_to_compute.compute_task_def['task_id'] = 1
+        task_to_compute.compute_task_def['deadline'] = int(dateutil.parser.parse("2017-12-01 11:00:00").timestamp())
```
Comment (kbeker) [inline concent_api/utils/tests/test_integration_api_view.py]: Style

File: concent_api/utils/tests/test_integration_api_view.py [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+
+        # STEP 1: Provider forces computed task report via Concent
+
+        task_to_compute = message.TaskToCompute(
+            timestamp   = int(dateutil.parser.parse("2017-12-01 10:00:00").timestamp()),
+        )
+
+        task_to_compute.compute_task_def = message.ComputeTaskDef()
+        task_to_compute.compute_task_def['task_id']     = 1
+        task_to_compute.compute_task_def['deadline']    = int(dateutil.parser.parse("2017-12-01 11:00:00").timestamp())
```
Comment (kbeker) [inline concent_api/utils/tests/test_integration_api_view.py]: Style

**Comment by Jakub89** [comment]
Style fixed and split to two commits.

**Comment by Jakub89** [comment]
@rwrzesien  I've done all the steps, squash fixes, and change the commit name, because "small style fixes" was't suitable anymore.

**Comment by Jakub89** [comment]
Done.

**Pull request merged**